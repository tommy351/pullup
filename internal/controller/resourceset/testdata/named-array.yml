---
apiVersion: pullup.dev/v1alpha1
kind: ResourceSet
metadata:
  name: test-46
  namespace: test
spec:
  number: 46
  resources:
    - apiVersion: apps/v1
      kind: Deployment
      metadata:
        name: test
      spec:
        template:
          spec:
            containers:
              - name: foo
                image: gcr.io/test/foo:v2
                env:
                  - name: B
                    value: b2
                  - name: C
                    value: c1
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test
  namespace: test
spec:
  selector:
    matchLabels:
      app: test
  template:
    metadata:
      labels:
        app: test
    spec:
      containers:
        - name: foo
          image: gcr.io/test/foo
          env:
            - name: A
              value: a1
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test-46
  namespace: test
  ownerReferences:
    - apiVersion: pullup.dev/v1alpha1
      kind: ResourceSet
      name: test-46
      controller: true
      blockOwnerDeletion: true
spec:
  selector:
    matchLabels:
      app: test
  template:
    metadata:
      labels:
        app: test
    spec:
      containers:
        - name: foo
          image: gcr.io/test/foo:v1
          env:
            - name: B
              value: b1
            - name: A
              value: a2
