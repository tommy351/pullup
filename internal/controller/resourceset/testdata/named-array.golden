- apiVersion: apps/v1
  kind: Deployment
  metadata:
    generation: 2
    labels:
      app: test
    name: test-46
    namespace: test
    ownerReferences:
    - apiVersion: pullup.dev/v1alpha1
      blockOwnerDeletion: true
      controller: true
      kind: ResourceSet
      name: test-46
      uid: ""
    selfLink: /apis/apps/v1/namespaces/test/deployments/test-46
  spec:
    progressDeadlineSeconds: 600
    replicas: 1
    revisionHistoryLimit: 10
    selector:
      matchLabels:
        app: test
    strategy:
      rollingUpdate:
        maxSurge: 25%
        maxUnavailable: 25%
      type: RollingUpdate
    template:
      metadata:
        creationTimestamp: null
        labels:
          app: test
      spec:
        containers:
        - env:
          - name: A
            value: a2
          - name: B
            value: b2
          - name: C
            value: c1
          image: gcr.io/test/foo:v2
          imagePullPolicy: IfNotPresent
          name: foo
          resources: {}
          terminationMessagePath: /dev/termination-log
          terminationMessagePolicy: File
        dnsPolicy: ClusterFirst
        restartPolicy: Always
        schedulerName: default-scheduler
        securityContext: {}
        terminationGracePeriodSeconds: 30
  status: {}
